### PR TITLE
Add specialist slot tooltip for buildings that provide them

### PR DESCRIPTION
**Pull request by TeagueLander**
#1762

### CODE REVIEW

File: android/assets/jsons/translationsByLanguage/template.properties [hunk 1]
```diff
@@ -819,7 +819,17 @@ Nothing =
...
+[specialist] slots = 
+Engineer specialist = 
+Merchant specialist = 
+Scientist specialist = 
+Artist specialist = 
+[specialist] slots = 
+Engineer specialist = 
+Merchant specialist = 
+Scientist specialist = 
+Artist specialist =
```
Comment (r3versi) [inline android/assets/jsons/translationsByLanguage/template.properties]: Duplicates?

Comment (TeagueLander) [reply android/assets/jsons/translationsByLanguage/template.properties]: Fixed.

**Comment by yairm210** [comment]
Can you attach a screenshot of how it looks in-game?

**Comment by TeagueLander** [comment]
> Can you attach a screenshot of how it looks in-game?

image

Note: it is NOT displayed on the technology screen.  This is because I was trying to copy how greatPersonPoints are displayed.

image

**Comment by yairm210** [comment]
Looks good!

**Pull request merged**